content/docs/usage/setup/access.mdx

```diff
@@ -2,4 +2,51 @@
 title: Getting Access
 ---
 
-Coming Soon.
+## T6 Hardware
+### With a monitor and keyboard available
+
+- Insert the microSD card with image flashed.
+- Ensure you have an ethernet cable plugged into your T6 box.
+- Plug in a HDMI screen, make sure it's plugged into HDMI1.
+- Plug in a keyboard.
+- Plug power in.
+- Wait until the box finishes booting, this could take anywhere from 2 to 10 minutes.
+- Once finished, login with the username `shibe` and the password `suchpass`. You will be prompted to change the password.
+- Enter command `ip addr` and take note of your IP address.
+- Ignore the `lo: <LOOPBACK>` section, look for another one.
+- Look for a line that starts with inet followed by your IP address, in this example 192.168.1.5.
```

Comment:
Maybe also:

Look for an IP address that starts with either `10.` or `192.168.` or `172.`?